### PR TITLE
Create Pipewire topic

### PR DESCRIPTION
**Pull request by yavko**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/master/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/master/docs>

> Pipewire needs to have its own topic, because its getting popular, and popular things need their own topic.

---

### CODE REVIEW

**Review by kenyonj — APPROVED** [review]
Thanks @yavko! This is great. I'll get it merged in as soon as CI is green (going to kick that process off now).

File: topics/pipewire/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+display_name: Pipewire
+short_description: PipeWire is a server for handling audio and video streams and hardware on Linux.
+topic: pipewire
+wikipedia_url: https://en.wikipedia.org/wiki/PipeWire
```
[the remark this answers is not in the document]
Comment (yavko) [reply topics/pipewire/index.md:5]: ok yeah that needs to be added

**Comment by yavko** [comment]
Oops i accidentally made a review

**Comment by kenyonj** [comment]
No problem, the change is live: github.com/topics/pipewire!

**Comment by yavko** [comment]
> No problem, the change is live: [github.com/topics/pipewire](https://github.com/topics/pipewire) !

Sounds great!

**Pull request merged**